wandb/sdk/artifacts/artifact.py

```diff
@@ -1713,7 +1714,7 @@ def _run_artifact_download(
     ) -> FilePathStr:
         assert wandb.run is not None, "failed to initialize run"
         run = wandb.run
-        if run._settings._require_nexus:
+        if get_core_path():
```

Comment:
Nit: it's no longer clear what this check does, let's add a comment:
```suggestion
        if get_core_path():  # require Nexus
```

Reply:
good idea!